### PR TITLE
rqt_pose_view Quaternion support

### PR DESCRIPTION
**Pull request by martonmiklos**
Added support for displaying Quaternion type in the rqt_pose_view.

### CODE REVIEW

File: rqt_pose_view/src/rqt_pose_view/pose_view_widget.py [hunk 1]
```diff
@@ -39,6 +39,7 @@
 import rospy
 from rostopic import get_topic_class
 from tf.transformations import quaternion_matrix, quaternion_about_axis
+from geometry_msgs.msg import Quaternion
```
Comment (DorianScholz) [inline rqt_pose_view/src/rqt_pose_view/pose_view_widget.py]: As far as I can see this import is never used.

File: rqt_pose_view/src/rqt_pose_view/pose_view_widget.py [hunk 2]
```diff
@@ -102,9 +103,13 @@ def _set_default_view(self):
         self._gl_view.rotate((1, 0, 0), -45)
         self._gl_view.translate((0, -3, -15))
 
-    def message_callback(self, message):
+    def message_callback_pose(self, message):
         self._position = (message.position.x, message.position.y, message.position.z)
         self._orientation = (message.orientation.x, message.orientation.y, message.orientation.z, message.orientation.w)
+            
+    def message_callback_quaternion(self, message):
+        self._position = (0,0,0)
```
Comment (DorianScholz) [inline rqt_pose_view/src/rqt_pose_view/pose_view_widget.py]: Please add spaces after the commata.

**Comment by DorianScholz** [comment]
Thanks for the pull request. 
Lgtm, except for my minor remarks above.

**Comment by martonmiklos** [comment]
Fixed the mentioned issues, thank you very much for your review!

**Comment by DorianScholz** [comment]
Thanks for the fixes!

**Pull request merged**